inginious/agent/docker_agent/__init__.py

```diff
@@ -422,8 +425,8 @@ def __new_job_sync(self, message, future_results):
                                                                                      hard_time_limit, share_network, write_stream))
                             elif msg["type"] == "ssh_key":
                                 # send the data to the backend (and client)
-                                self._logger.info("%s %s", self._running_ssh_debug[container_id], str(msg))
-                                await self.send_ssh_job_info(job_id, self._ssh_host, ssh_port, msg["ssh_key"])
+                                self._logger.info("%s %s", container_id, str(msg))
+                                await self.send_ssh_job_info(job_id, self._address_host, ports[22], msg["ssh_key"])
```

Comment:
@anthonygego shouldn't we directly update the "ssh_key" message code to be able to set another port?

Reply:
That is not critical but could be nice.